GittieLabs/lovebank-frontend
# Dev login testing

To test:
-ensure that firebase mock works (used in the context of widget testing)
-ensure that test suite passes (run the `flutter test` command on the command line)
-ensure that test/widget_test.dart tests the login/register capability to satisfaction
-keep an eye out for assorted bugs
